source/plugin/scheduler.rst

```diff
@@ -30,6 +31,26 @@ which you can specify using ``Task.Builder#execute(Runnable runnable)``:
         }
     });
 
+or using Java 8 syntax with ``Task.Builder#execute(Runnable runnable)``
+
+.. code-block:: java
+
+    taskBuilder.execute(
+        () -> {
+            logger.info("Yay! Schedulers!");
+        }
+    );
+
+or using Java 8 syntax with ``Task.Builder#execute(Consumer<Task> task)``
```

Comment:
Move this with the `Consumer<Task>` example. Also please change the example so that both consumer examples do the same thing.